src/encoded/static/scss/encoded/_layout.scss

```diff
@@ -583,15 +583,41 @@ $margin-util-sizes-mini: 1, 2, 3, 4, 5, 6, 7, 8; 		// * 1px
 
 .modal-dialog {
 
```

Comment:
Performed minor CSS edits and improved 'targeting' in CSS selector strings so that these styles (including that in `../bootstrap/_modals.scss` above) are prioritized over the overrides [defined in HiGlassComponent/hglib.css](https://github.com/higlass/higlass/blob/a44dcf9226f0e91275ad1ce01f8accf1340a448f/app/styles/TrackOptions.css#L30-L37).

Added [related issue](https://github.com/higlass/higlass/issues/638) to higlass repo.